apps/widget-e2e/src/describer/modelAssessment/Constants.ts

```diff
@@ -21,5 +21,15 @@ export enum Locators {
   ICEXAxisNewValue = "#subPlotContainer text[class^='xtitle']",
   ICEToolTipButton = "#subPlotContainer button:contains('How to read this chart')",
   ICECalloutTitle = "#subPlotContainer div.ms-Callout-container span[class^='calloutTitle']",
-  ICECalloutBody = "#subPlotContainer div.ms-Callout-container div[class^='calloutInner']"
+  ICECalloutBody = "#subPlotContainer div.ms-Callout-container div[class^='calloutInner']",
+  SortByDropdown = "div[class^='globalChartWithLegend'] div.ms-Dropdown-container",
```

Comment:
feel free to add id to the elements you want to test in e2e.

Reply:
thank you, going forward will add IDs to help locate elements.